packages/fx-core/src/plugins/resource/apiconnector/plugin.ts

```diff
@@ -444,10 +448,12 @@ export class ApiConnectorImpl {
     return node;
   }
 
-  public exactTelemetryProperties(config: ApiConnectorConfiguration): { [key: string]: string } {
+  public getTelemetryProperties(config: ApiConnectorConfiguration): { [key: string]: string } {
     const properties = {
       [Telemetry.properties.authType]: config.AuthConfig.AuthType.toString(),
+      [Telemetry.properties.componentPath]: config.ComponentPath.join(","),
```

Comment:
let's remove the path, since it may contain sensitive info

Reply:
we can add component type (bot/api) instead, since they're constant defined by us

Reply:
Here the path is api/bot, not the real file path.
Will update the property key to avoid confuse.